### ISSUE TITLE
[Feature Request] Timestamp

### ISSUE THREAD

**Issue by Laicure** [moezbhatti/qksms#1462]
### DESCRIPTION
This is a feature request coming from a user of YAATA sms app (that's me).

### Feature
Show Sent timestamp beside Received Timestamp when message is received later than 1 minute of being sent by the recipient:

(recipient's message bubble)
02:12pm -> 02:15pm

### OBSERVATIONS
Qksms only shows the Received timestamp.
User still needs to tap and show the details of the recipient's message to view the sent timestamp.

Please do tell me if you need further clarifications.
I really want this feature in your app.
I think those are just very small ifs in the code.

**Comment by moezbhatti** [comment]
Related to #1452

**Comment by sm4rk0** [comment]
But what if a message is sent on 2019-12-31 23:59 and received on 2020-01-01 00:00? That's not as easy to fit into UI as 04:20 -> 04:22. I recon it could be displayed as Tue 23:59 -> Wed 00:00, but still, there are more complicated cases.

**Comment by moezbhatti** [comment]
how about just the amount of time it was late by? then you can open the message details to see the full timestamp

**Comment by sm4rk0** [comment]
Good idea! And show only a timestamp if the delay is less than a set treshlod (one minute?).

**Comment by Laicure** [comment]
Thanks for looking into this, you may close this issue once implemented <3

**Comment by SebiderSushi** [comment]
Since the device usually fetches all delayed messages at once as soon as it reconnects to the mobile network, i can see another alternative:

QKSMS could desplay and use sent timestamps everywhere (including the tap-on-message timestamp) by default and enrich the "divider timestamp" that already seperates messages if more time passes to mark each block of messages that were delayed at least one minute as follows:

```
Received: [timestamp]
Sent: [timestamp of first message]
[message1]

Received: [timestamp]
Sent: [timestamp of first message]
[message1]
[message2]
            [reply1]

Received: [timestamp]
Sent: [timestamp of first message]
[message1]
            [reply1]
[message2]

[timestamp of non-delayed message]
[message1]
            [reply1]
[message2]
```
